scripts/review/diff.ts

```diff
@@ -32,6 +32,41 @@ function hasChangeTypes(changes: parseDiff.Chunk['changes'], types: string[]): b
   return changes.length === types.length && changes.every(({ type }, i) => type === types[i]);
 }
 
+function highestChainID(changes: parseDiff.Chunk['changes']) {
+  assert(hasChangeTypes(changes, ['del', 'add', 'add']));
+  assert(changes[0].content.replace(/^-(.*)/, '+$1,') === changes[1].content);
+}
```

Comment:
nit: it's unclear to me what this function does, I'm looking at the name and at the function body and can't find a correlation

Reply:
I have added some comments for more details.

Reply:
Thanks, it's much better now. I think if the function combines assertions, it'd better be named `assertHighestChainIDChangesValidity` or similar

Reply:
Done @mmv08 👍🏾